test/test_hydrostatic_free_surface_models.jl

```diff
@@ -210,24 +211,34 @@ topos_3d = ((Periodic, Periodic, Bounded),
                          HydrostaticSphericalCoriolis(scheme=EnstrophyConserving()))
 
             @testset "Time-stepping HydrostaticFreeSurfaceModels [$arch, $(typeof(coriolis))]" begin
-                @test time_step_hydrostatic_model_works(lat_lon_sector_grid, coriolis=coriolis)
-                @test time_step_hydrostatic_model_works(lat_lon_strip_grid, coriolis=coriolis)
+                @test time_step_hydrostatic_model_works(lat_lon_sector_grid; coriolis)
+                @test time_step_hydrostatic_model_works(lat_lon_strip_grid; coriolis)
             end
         end
 
         for momentum_advection in (VectorInvariant(), CenteredSecondOrder(), WENO())
             @testset "Time-stepping HydrostaticFreeSurfaceModels [$arch, $(typeof(momentum_advection))]" begin
                 @info "  Testing time-stepping HydrostaticFreeSurfaceModels [$arch, $(typeof(momentum_advection))]..."
-                @test time_step_hydrostatic_model_works(rectilinear_grid, momentum_advection=momentum_advection)
+                @test time_step_hydrostatic_model_works(rectilinear_grid; momentum_advection)
             end
         end
 
         momentum_advection = VectorInvariant()
         @testset "Time-stepping HydrostaticFreeSurfaceModels [$arch, $(typeof(momentum_advection))]" begin
             @info "  Testing time-stepping HydrostaticFreeSurfaceModels [$arch, $(typeof(momentum_advection))]..."
-            @test time_step_hydrostatic_model_works(lat_lon_sector_grid, momentum_advection=momentum_advection)
+            @test time_step_hydrostatic_model_works(lat_lon_sector_grid; momentum_advection)
         end
 
+        for tracer_advection in [WENO(),
+                                 FluxFormAdvection(WENO(), WENO(), Centered())
```

Comment:
minor comment: is there a reason why the specific addiction schemes were chosen? if so perhaps a comment is good?

Reply:
More cases can be added. This just spans three significantly different possibilities. Only the 3rd is broken currently

Reply:
All the other tests are the same btw. They are not exhaustive

Reply:
I didn't mean to imply we need to add more test. I was just wondering if there was a reason z was different than the x, y (and if so, perhaps a comment would be appropriate in that case). But if it's just randomness then no worries!